weed/server/volume_server_handlers.go

```diff
@@ -22,6 +22,11 @@ security settings:
 */
 
 func (vs *VolumeServer) privateStoreHandler(w http.ResponseWriter, r *http.Request) {
+	if r.Method == "GET" && r.URL.Path == "/" { // for convenience to /ui/index.html
```

Comment:
do not do this. not easy to manage access.

Reply:
Got.